nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/main/java/org/apache/nifi/processors/poi/ConvertExcelToCSVProcessor.java

```diff
@@ -101,6 +100,34 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor FIRST_ROW = new PropertyDescriptor
+            .Builder().name("excel-extract-first-row")
+            .displayName("First Row")
+            .description("The row number of the header row, or first row of data if `Has Header Line` is set to false. "
+                    + "Use this to skip over rows of data at the top of your worksheet that are not part of the dataset.")
+            .required(true)
+            .defaultValue("0")
+            .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor COLUMNS_TO_SKIP = new PropertyDescriptor
+            .Builder().name("excel-extract-column-to-skip")
+            .displayName("Columns To Skip")
+            .description("Comma delimited list of column numbers to skip. Use the columns number and not the letter designation. "
+                    + "Use this to skip over columns anywhere in your worksheet that you don't want extracted as part of the record.")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor FORMAT_VALUES = new PropertyDescriptor.Builder()
+            .name("excel-format-values")
+            .displayName("Format Cell Values")
+            .description("Should the cell values be written too CSV using the formatting applied in Excel, or should they be printed as raw values.")
+            .allowableValues("true", "false")
+            .defaultValue("true")
```

Comment:
Only to preserve existing flow behavior (it's possible that some user has been running a flow reading excel files with styles), `false` might be better as default value.